### PR TITLE
Update 0-.ssh.md

### PR DESCRIPTION
**Pull request by antongisli**
might need to verify this:
"Users can associate multiple SSH keys to a single project, however only one SSH key can be applied to a single cluster." also, seems that i couldn't set the ssh key for AWS?

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1139#" title="Author self-approved">antongisli</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @antongisli. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:19]:
```suggestion
Palette groups clusters for logical separation into [Projects](/projects). Users and teams can be assigned roles within a project for granular control over permissions within the project scope. SSH key authentication is scoped to a project. Multiple users can be given access to a single project. To gain access to a cluster with SSH, you need to have a public SSH key registered in Palette.
```

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:21]:
```suggestion
# Upload SSH Key in Palette

```

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:23]:
```suggestion
Before you begin, generate an SSH key pair. We recommend using the RSA (Rivest-Shamir-Adleman) public-key cryptographic system” to initiate a secure handshake between remote clusters for every cloud, data center, and edge environment supported by Palette.
```
Changing to the long form to help the reader.

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:25]:
```suggestion
The following steps guide you on how to upload your SSH key in Palette.
```

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:26]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).

```

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:27]:
```suggestion
2.  Navigate to the left **Main Menu** and select **Project Settings**. Next, select the **SSH Keys** tab.
```

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:28]:
```suggestion
3. Open the **+Add New SSH Key** tab to complete the **Add Key** form as below:
```
Using bold to indicate UI element.

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:33]:
```suggestion
<InfoBox>

SSH keys can be edited and deleted later by using the **three-dot Menu** to the right of each key.

</InfoBox>
```

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:31]:
```suggestion
4. Click **Confirm** to complete the wizard.

<br />
```

File: content/docs/04-clusters/06-cluster-management/0-.ssh.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/0-.ssh.md:35]:
```suggestion
# Use SSH Key

```

**Review by karl-cardenas-coding — APPROVED** [review]
Thank you for tackling this one @antongisli . Just some minor suggestions from the style guide.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1139#" title="Author self-approved">antongisli</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1139#pullrequestreview-1304127583" title="Approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
